packages/flutter/lib/src/material/drawer.dart

```diff
@@ -83,6 +85,7 @@ class Drawer extends StatelessWidget {
   /// Typically used in the [Scaffold.drawer] property.
   const Drawer({
     Key key,
+    this.routeName = _kDefaultDrawerRouteNameSemantics,
```

Comment:
we should call this something like "semanticLabel" or "semanticTitle" or just "title".
People using a drawer aren't going to really understand that it has anything to do with routes.

Reply:
I've removed this property from drawer - can be specified by having a child semantics node with a route/value.  For this and the other widgets, I've added default semantics which have a route flag and no name.

Maybe we can have some sensible defaults (maybe not) or provide a nicer way to give a semantic name to this route.  At any rate, it needs more discussion and doesn't have to happen now.